CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [ENHANCEMENT] HelmCharts: You can now configure `storageClass` per zone for Alertmanager, StoreGateway and Ingester.  
```

Comment:
Can you please move this to the helm-specific changelog and add the PR number? 
https://github.com/grafana/mimir/blob/6979e849ccb2c39dcbcb143a67f874e4b6a30b11/operations/helm/charts/mimir-distributed/CHANGELOG.md#L29

```suggestion
* [ENHANCEMENT] HelmCharts: You can now configure `storageClass` per zone for Alertmanager, StoreGateway and Ingester. #4234
```